product_code/matriculation_service/api/src/main/scala/de/upb/cs/uc4/matriculation/api/MatriculationService.scala

```diff
@@ -39,7 +40,7 @@ trait MatriculationService extends UC4HyperledgerService {
         restCall(Method.GET, pathPrefix + "/matriculation/:username", getMatriculationData _),
 
         restCall(Method.OPTIONS, pathPrefix + "/matriculation/:username/unsigned_proposal", allowedPost _),
-        restCall(Method.OPTIONS, pathPrefix + "/matriculation/:username", allowedGet _)
+        restCall(Method.OPTIONS, pathPrefix + "/matriculation/:username", allowedGet _),
```

Comment:
```suggestion
        restCall(Method.OPTIONS, pathPrefix + "/matriculation/:username", allowedGet _)
```
why a comma at the end of the last line?